featuretools/primitives/standard/transform/postal/two_digit_postal_code.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Categorical, PostalCode
+
+from featuretools.primitives.base import TransformPrimitive
+from featuretools.utils.gen_utils import Library
+
+
+class TwoDigitPostalCode(TransformPrimitive):
+    """Returns the two digit prefix from a given postal code.
+
+    Description:
+        For a list of postal codes, returns the two digit prefix for a given postal code.
+
+    Examples:
+        >>> two_digit_postal_code = TwoDigitPostalCode()
+        >>> two_digit_postal_code(['92432', '34514']).tolist()
+        ['92', '34']
+    """
+
+    name = "two_digit_postal_code"
+    input_types = [ColumnSchema(logical_type=PostalCode)]
+    compatibility = [Library.PANDAS, Library.DASK, Library.SPARK]
+    return_type = ColumnSchema(logical_type=Categorical, semantic_tags={"category"})
+    description_template = "The two digit postal code prefix of {}"
+
+    def get_function(self):
+        def two_digit_postal_code(postal_codes):
+            def transform_postal_code(pc):
+                return str(pc)[:2] if not pd.isna(pc) else pc
+
+            return postal_codes.apply(transform_postal_code)
```

Comment:
An alternate approach here, which might simplify the code is to use `.str.slice` to get the values. So instead of using `.apply(transform_postal_code)` you could just return `postal_codes.str.slice(start=0, stop=2)` and eliminate the need to define the custom function for `.apply`.

Not sure if there is a performance advantage of one over the other, but there should be less code with `.str.slice` assuming it works in all cases (might not work for the integer categorical edge case, but I'm not 100% convinced we will should cover that yet).

Reply:
I will run some perf tests and get back to you on this.

Reply:
It looks like performance is nearly identical